src/charm.py

```diff
@@ -176,9 +137,26 @@ def _check_model_name(self):
 
     def _check_leader(self):
         if not self.unit.is_leader():
-            # We can't do anything useful when not the leader, so do nothing.
             raise CheckFailed("Waiting for leadership", WaitingStatus)
 
+    def _update_layer(self, profiles_service: str) -> None:
+        """Updates the Pebble configuration layer if changed."""
+        current_layer = self._container.get_plan()
+        new_layer = self.get_kubeflow_dashboard_operator_layer(profiles_service)
+        self.logger.info(f"NEW LAYER: {new_layer}")
```

Comment:
Add a TODO so you don't forget to remove later?

Reply:
When what ? sorry I dont get it :D